simulation/state.go

```diff
@@ -190,6 +249,39 @@ func updateAuthorityState(
 	return authorityState
 }
 
+// updateCrossChainState updates the crosschain genesis state.
+// It adds the following
+// - A gas price list for each chain
+func updateCrossChainState(
```

Comment:
```suggestion
func updateCrosschainState(
```
Name of the module is one word